internal/table/table.go

```diff
@@ -137,3 +164,35 @@ func withColor(c *color.Color, f func()) {
 	f()
 	color.Unset()
 }
+
+func splitLines(text string, maxWidth int) lines {
+	// just a precaution to not allocate the universe
+	if maxWidth > 65535 {
+		maxWidth = 65535
+	}
+	if maxWidth == 0 {
+		panic("splitLines: maxWidth cannot be 0")
```

Comment:
Should we panic?

Reply:
`maxWidth` should never be `0` in the normal operation, that would be an exceptional case.